lib/time/time.go

```diff
@@ -0,0 +1,548 @@
+package time
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+	"go.starlark.net/syntax"
+)
+
+// ModuleName defines the expected name for this Module when used in the
+// starlark runtime.
+const ModuleName = "time"
+
+// Module time is a Starlark module of time-related functions.
+var Module = &starlarkstruct.Module{
+	Name: ModuleName,
+	Members: starlark.StringDict{
+		"duration":       starlark.NewBuiltin("duration", newDuration),
+		"parse_duration": starlark.NewBuiltin("parse_duration", parseDuration),
+		"parse_location": starlark.NewBuiltin("parse_location", parseLocation),
+		"now":            starlark.NewBuiltin("now", now),
+		"struct":         starlark.NewBuiltin("struct", starlarkstruct.Make),
+		"time":           starlark.NewBuiltin("time", newTime),
+		"parse_time":     starlark.NewBuiltin("time", parseTime),
+		"sleep":          starlark.NewBuiltin("sleep", sleep),
+		"from_timestamp": starlark.NewBuiltin("fromtimestamp", fromUnixTime),
```

Comment:
Maybe you should name the called function `fromTimestamp` instead of `fromUnixTime` to keep the naming pattern.

Reply:
Please mark as resolved.